src/components/snapshots/routes/routeUploadSnapshot.ts

```diff
@@ -8,9 +8,9 @@ import {
 import fs from 'fs'
 import { pipeline } from 'stream'
 import { promisify } from 'util'
-import zipper from 'adm-zip'
 import { timestampStringExpression } from './helpers'
 import { DateTime } from 'luxon'
+import StreamZip = require('node-stream-zip')
```

Comment:
```suggestion
import StreamZip from 'node-stream-zip'
```
Better to not use `require` any more.